scripting/pugsetup/captainpickmenus.sp

```diff
@@ -4,7 +4,10 @@ int g_PickCounter = 0;
 
 public Action Timer_InitialChoiceMenu(Handle timer) {
   int client = g_capt1;
-
+  
+  if(!IsClientInGame(client))
+    return Plugin_Handled;
```

Comment:
I suspect if this is reached, the game should be ended, otherwise it's left in a akward state that requires force-ending.

I'd suggest something similar to how a missing captain is handled around lines 115:

```
      PugSetup_MessageToAll("A captain is missing, aborting the game.");
      EndMatch(false);    
```

Reply:
thanks for the suggestion, I have added that to the next commit , please check again.